packages/upswyng-server/src/routes/index.svelte

```diff
@@ -12,63 +12,64 @@
     width: 100%;
     margin-right: 6em;
   }
-
-  .hero-container {
-    display: flex;
-    align-items: center;
-  }
-
-  .hero-image-container {
-    flex: 3;
-    margin-right: 3em;
-  }
-
-  .hero-container p {
-    flex: 2;
-  }
-
-  .hero-tag {
-    color: white;
-    line-height: 1.25em;
-  }
-
-  .visit-web-app {
-    display: block;
-    margin: 0 auto;
-  }
 </style>
 
 `
 <svelte:head>
-  <title>Upswyng</title>
+  <title>Upswyng: Provider Portal</title>
 </svelte:head>
 
-<section class="hero">
+<section class="hero is-dark">
   <div class="hero-body upswyng-intro">
-    <div class="container hero-container">
-      <div class="hero-image-container">
-        <img alt="Upswyng: Resource within reach" src="upswyng_light.svg" />
+    <div class="container">
+      <div class="columns is-variable is-6 is-vcentered">
+        <div class="column">
+          <img alt="upswyng: Resources within reach" src="upswyng_light.svg" />
+        </div>
+        <div class="column">
+          <h1 class="title">Upswyng Provider Portal</h1>
```

Comment:
We need to decide if this is UpSwyng or Upswyng and also the Provider's Portal or Provider Portal